iina/FFmpegController.m

```diff
@@ -615,19 +615,13 @@ + (NSImage *)createNSImageWithContentsOfURL:(nonnull NSURL *)url
       case AVCOL_PRI_BT2020:
         if (@available(macOS 11.0, *)) {
           cgColorSpace = CGColorSpaceCreateWithName(kCGColorSpaceITUR_2100_PQ);
-        } else if (@available(macOS 10.15.4, *)) {
-            cgColorSpace = CGColorSpaceCreateWithName(kCGColorSpaceITUR_2020_PQ);
-        } else if (@available(macOS 10.14.6, *)) {
-            cgColorSpace = CGColorSpaceCreateWithName(kCGColorSpaceITUR_2020_PQ_EOTF);
```

Comment:
This code **must not** be removed.

Thank you very much for working to reduce the deprecated warnings. The IINA project considers it important to fix compiler warnings. Your help is appreciated. Unfortunately **Xcode is BROKEN** and can trick you into removing code that is needed. The file `Configs/Deployment.xconfig` contains:
```text
MACOSX_DEPLOYMENT_TARGET = 10.15
MACOSX_DEPLOYMENT_TARGET[arch=arm64] = 12
```

This has confused Xcode and it is emitting deprecated messages as if the minimum macOS version for IINA is macOS Monterey. It is not. The minimum macOS version is macOS Catalina. If you are building on a Mac with a M1 chip in Xcode change your build destination to `My Mac (Rosetta)` then clean and rebuild. You will see many of these deprecated warnings disappear. To do that you may have to enable the Rosetta destination. Apple is pushing developers away from Rosetta by hiding that destination in Xcode by default. If you have trouble enabling Rosetta another way to see this is to change the above file to use `10.15` for `arch=arm64` then clean and rebuild.